calour/io.py

```diff
@@ -393,36 +414,99 @@ def read_amplicon(data_file, sample_metadata_file=None,
 
 
 @ds.with_indent(4)
-def read_open_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_metadata_file=None,
-                 description=None, sparse=False, rows_are_samples=False, mz_rt_sep=None, *, normalize, **kwargs):
-    '''Load an OpenMS metabolomics experiment.
+def read_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_metadata_file=None,
+            file_type='mzmine2', rows_are_samples=None, table_sep=None, mz_rt_sep='_', direct_ids=None,
```

Comment:
to be consistent with `read`, `file_type` -> `data_file_type`

Reply:
fixed